setup.py

```diff
@@ -57,21 +40,21 @@
         'invenio-db[versioning]>=1.0.0',
     ],
     'records': [
-        'invenio-deposit>=1.0.0a7',
-        'invenio-records>=1.0.0b1',
+        'invenio-deposit>=1.0.0',
```

Comment:
Some modules are not 1.0.0 unfortunately (I think it's only `invenio-deposit` though)

Reply:
It seems `change_license.py` stripped all the alphas and betas without checking. Should I keep the old alphas/betas that works or should I try to use the non-betas/alphas as minimum requirements whenever possible?